src/main/java/com/thealgorithms/datastructures/graphs/README.md

```diff
@@ -1,6 +1,6 @@
-## Graph
+## Graphs
 
-Graph is a useful data structure for representing most of the real world problems involving a set of users/candidates/nodes and their relations. A Graph consists of two parameters :
+"Graph is a useful data structure for representing most of the real-world problems involving a set of users/candidates/nodes and their relations. A graph consists of two parameters:
```

Comment:
What is this "?

Reply:
In simple terms, a "graph" is a helpful way to represent real-world problems where you have a group of things (like people, candidates, or nodes) and connections between them.
Imagine you have a bunch of friends, and you want to show who is friends with whom. You could use a "graph" to do this. In a graph, there are two main things to consider:
Nodes: These are like the individuals in your group, like your friends. Each person is a node.
Relations: These are the connections between people, like who is friends with who. In a graph, we represent these connections with lines or arrows between nodes.
So, a graph helps you organize information about a group of things and how they are connected. It's a useful tool for solving problems involving relationships between different items or people.

Reply:
I'm talking about the double quote in the very beginning of the line that doesn't seem to serve any purpose there